decoder/weather.go

```diff
@@ -30,7 +30,7 @@ type Weather struct {
 	SpecialEffectLevel null.Int  `db:"special_effect_level"`
 	Severity           null.Int  `db:"severity"`
 	WarnWeather        null.Bool `db:"warn_weather"`
-	Updated            int64     `db:"updated"`
+	UpdatedMs          int64     `db:"updated"`
```

Comment:
why this change? you multiply and divide with 1000 multiple times?! just multiply with thousand when you need to compare it with the GMO timestamp?

Reply:
If I recall this is to ensure strict ordering within the same second.

Reply:
This fixes race conditions ensuring that only earlier GMO would never override latest GMO data. This would also prevent weather cell switching back and forth which might happen more frequently if multiple workers are in the same cell.

On hindsight, this is clearly the right thing to do since the beginning. Trimming the millisecond literally saves nothing in the database and wastes precious **server-side** CPU cycles for division. I would personally suggest to even change the entire data structure in the database (for weather and all else like gyms pokestops) and have `updated` as a computed column going forward in a separate PR.

Reply:
> On hindsight, this is clearly the right thing to do since the beginning. Trimming the millisecond literally saves nothing in the database and wastes precious **server-side** CPU cycles for division.

The trimming of the time for the database is just a reflection of copying the RDM schema.  It doesn't seem worth the downstream impact of changing it now - an integer division will be nowhere on the chart compared to pretty much any other processing element